### PR TITLE
Revert "implement drop subscription in 3 requests"

### PR DESCRIPTION
**Pull request by BaurzhanSakhariev**
This reverts part of the commit [562abc8a](https://github.com/crate/crate/commit/6857a28b669b6e6f056e59cbfdc760c59739701c) related to close request logic 
(and keeps splitting ALTER_TABLE_OPEN_CLOSE into separate OPEN and CLOSE operations) 
and also reverts https://github.com/crate/crate/commit/562abc8a358c344e3986cc1f1a27e06cec65f43c

I kept changes from https://github.com/crate/crate/pull/12280/, https://github.com/crate/crate/pull/12370,  https://github.com/crate/crate/pull/12378, https://github.com/crate/crate/pull/12220/ and https://github.com/crate/crate/pull/12231

## Checklist

 - [ ] Added an entry in `CHANGES.txt` for user facing changes
 - [ ] Updated documentation & `sql_features` table for user facing changes
 - [ ] Touched code is covered by tests
 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [ ] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

**Comment by BaurzhanSakhariev** [comment]
not related failure (not reproducible locally) 
REPRODUCE WITH: `./gradlew :server:test -Dtests.seed=1B103F162F79538F -Dtests.class=io.crate.integrationtests.DiskUsagesITest -Dtests.method="testRerouteOccursOnDiskPassingHighWatermark" -Dtests.locale=ses-Latn-ML -Dtests.timezone=Europe/Brussels`

```
java.lang.AssertionError: node2 has at least 3 shards
Expected: a value equal to or greater than <3>
     but: <0> was less than <3>
```

retest this please

File: server/src/main/java/io/crate/execution/ddl/tables/AlterTableOperation.java [hunk 1]
```diff
@@ -143,25 +143,22 @@ private Session session() {
...
         }
         if (openTable || clusterService.state().getNodes().getMinNodeVersion().before(Version.V_4_3_0)) {
             OpenCloseTableOrPartitionRequest request = new OpenCloseTableOrPartitionRequest(
-                tables,
-                partitionIndexName,
-                openTable,
-                ignoreUnavailableIndices
-            );
+                tableInfo.ident(), partitionIndexName, openTable);
             return transportOpenCloseTableOrPartitionAction.execute(request, r -> -1L);
```
Comment (BaurzhanSakhariev) [inline server/src/main/java/io/crate/execution/ddl/tables/AlterTableOperation.java:156]: `return transportOpenCloseTableOrPartitionAction.execute(request, r -> -1L) `is kept from https://github.com/crate/crate/pull/12280

File: server/src/main/java/io/crate/execution/ddl/tables/TransportOpenCloseTableOrPartitionAction.java [hunk 2]
```diff
@@ -79,11 +84,10 @@ public ClusterStateTaskExecutor<OpenCloseTableOrPartitionRequest> clusterStateTa
 
     @Override
     protected ClusterBlockException checkBlock(OpenCloseTableOrPartitionRequest request, ClusterState state) {
-        return TransportCloseTable.checkBlock(
-            state,
-            request.ignoreUnavailableIndices(),
-            request.tables(),
-            request.partitionIndexName()
-        );
+        if (isEmptyPartitionedTable(request.tableIdent(), state)) {
```
Comment (BaurzhanSakhariev) [inline server/src/main/java/io/crate/execution/ddl/tables/TransportOpenCloseTableOrPartitionAction.java:87]: this reverted to https://github.com/crate/crate/pull/12231 - first change before https://github.com/crate/crate/commit/6857a28b669b6e6f056e59cbfdc760c59739701c and https://github.com/crate/crate/commit/562abc8a358c344e3986cc1f1a27e06cec65f43c + kept changes from https://github.com/crate/crate/pull/12370 and https://github.com/crate/crate/pull/12378

File: server/src/main/java/io/crate/execution/ddl/tables/TransportCloseTable.java [hunk 3]
```diff
@@ -74,24 +74,21 @@
...
 import java.util.HashSet;
-import java.util.List;
 import java.util.Map;
 import java.util.Set;
 import java.util.function.Consumer;
-import java.util.stream.Collectors;
 
 public final class TransportCloseTable extends TransportMasterNodeAction<CloseTableRequest, AcknowledgedResponse> {
 
     private static final Logger LOGGER = LogManager.getLogger(TransportCloseTable.class);
```
Comment (BaurzhanSakhariev) [inline server/src/main/java/io/crate/execution/ddl/tables/TransportCloseTable.java:89]: same as for `TransportOpenCloseTableOrPartitionAction`:

 reverted to https://github.com/crate/crate/pull/12231 - first change before https://github.com/crate/crate/commit/6857a28b669b6e6f056e59cbfdc760c59739701c and https://github.com/crate/crate/commit/562abc8a358c344e3986cc1f1a27e06cec65f43c + kept changes from https://github.com/crate/crate/pull/12370 and https://github.com/crate/crate/pull/12378

**Review by seut — APPROVED** [review]
Lgtm otherwise

**Comment by BaurzhanSakhariev** [comment]
retest this please

**Pull request merged**